supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/GenomicsFactory.scala

```diff
@@ -88,15 +88,18 @@ case class GenomicsFactory(applicationName: String, authMode: GoogleAuthMode, en
 
       def networkFromLabels(vpcConfig: VirtualPrivateCloudConfiguration, projectLabels: ProjectLabels): Network = {
         val networkLabelOption = projectLabels.labels.find(l => l._1.equals(vpcConfig.name))
+        val subnetworkLabelOption = vpcConfig.subnetwork.flatMap(s => projectLabels.labels.find(l => l._1.equals(s)))
 
-        networkLabelOption match {
-          case Some(networkLabel) =>
-            new Network()
+        (networkLabelOption, subnetworkLabelOption) match {
+          case (Some(networkLabel), _) =>
+            val network = new Network()
               .setUsePrivateAddress(createPipelineParameters.runtimeAttributes.noAddress)
               .setName(VirtualPrivateCloudNetworkPath.format(createPipelineParameters.projectId, networkLabel._2))
-          case None =>
-            jobLogger.debug("Project `{}` does not have high security network configured. " +
-              "Project metadata does not have network label key `{}`. Falling back to running the job on default network.", createPipelineParameters.projectId: Any, vpcConfig.name: Any)
+
+            subnetworkLabelOption foreach { case(_, subnet) => network.setSubnetwork(subnet) }
+            network
+          case (None, _) =>
+            // Falling back to running the job on default network since the project does not have high security network configured
```

Comment:
TOL: Technically, ... `"... since the project does not provide the network specifying keys in its metadata"`.

In one project the network specified may not be high security, and in another the default network might be high security.

Reply:
>In one project the network specified may not be high security, and in another the default network might be high security.

I am not sure if I am completely following this. So are you saying that someone's `network-label-key` might point to a default network rather than a high security one?

Reply:
I'm just saying that "custom network" does not imply high security, and also that "default network" does not necessarily imply low security

Reply:
Ah I see. I will edit that comment.